main.tf

```diff
@@ -298,13 +298,12 @@ resource "spacelift_stack" "default" {
   before_perform                   = compact(coalesce(try(local.stack_configs[each.key].before_perform, []), var.before_perform))
   before_plan                      = compact(coalesce(try(local.stack_configs[each.key].before_plan, []), var.before_plan))
   branch                           = try(local.stack_configs[each.key].branch, var.branch)
-  description                      = coalesce(try(local.stack_configs[each.key].description, null), var.description)
   enable_local_preview             = try(local.stack_configs[each.key].enable_local_preview, var.enable_local_preview)
   enable_well_known_secret_masking = try(local.stack_configs[each.key].enable_well_known_secret_masking, var.enable_well_known_secret_masking)
   github_action_deploy             = try(local.stack_configs[each.key].github_action_deploy, var.github_action_deploy)
   labels                           = local.labels[each.key]
   manage_state                     = try(local.stack_configs[each.key].manage_state, var.manage_state)
-  name                             = each.key
+  name                             = local.configs[each.key].root_module
```

Comment:
Hm, I don't get this change. For multi-instance case we have one root module per multiple workspaces, and this is not a unique value.
Mind elaborating?

Reply:
You're correct! I could've just introduced a nasty bug... And honestly, we should have a test that avoids this. 

I made this change as I thought it looked cleaner, but it's totally a bug. Thank you for the catch!!!! 💯 

Will follow up with a fix + test.

Reply:
Fixed -- Thanks again @gberenice!